repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/services/httpconnectionpool/HttpsClientAuthMultiStorePoolTest.groovy

```diff
@@ -0,0 +1,156 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package features.services.httpconnectionpool
+
+import framework.ReposeValveTest
+import framework.mocks.MockIdentityV2Service
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.impl.client.DefaultHttpClient
+import org.eclipse.jetty.http.HttpVersion
+import org.eclipse.jetty.server.*
+import org.eclipse.jetty.server.handler.AbstractHandler
+import org.eclipse.jetty.util.ssl.SslContextFactory
+import org.rackspace.deproxy.Deproxy
+import spock.lang.Shared
+
+import javax.servlet.ServletException
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+import java.nio.file.Files
+import java.util.concurrent.TimeUnit
+
+class HttpsClientAuthMultiStorePoolTest extends ReposeValveTest {
+
+    @Shared
+    def Server server
+    @Shared
+    def File serverFile
+    @Shared
+    def File clientFile
+    @Shared
+    def statusCode = HttpServletResponse.SC_OK
+    @Shared
+    def responseContent = "The is the plain text test body data.\n".bytes
+    @Shared
+    def contentType = "text/plain;charset=utf-8"
+    @Shared
+    def identityEndpoint
+    @Shared
+    def MockIdentityV2Service fakeIdentityV2Service
+
+    def setupSpec() {
+        deproxy = new Deproxy()
+
+        reposeLogSearch.cleanLog()
```

Comment:
Not a bad thought. It is something that is done for just about every test that uses this framework. Also creating the `Deproxy` object would probably be good too.
We should probably add a story to clean this up, or just add it to the one to parallelize the framework.